dripline/implementations/ethernet_scpi_provider.py

```diff
@@ -0,0 +1,229 @@
+import re
+import socket
+import threading
+
+import scarab
+
+from dripline.core import Service
+# Dripline Exceptions currently unavailable
+
+# logging currently unavailable
+# import logging
+# logger = logging.getLogger(__name__)
+
+__all__ = []
+
+
+__all__.append('EthernetSCPIProvider')
+class EthernetSCPIProvider(Service):
```

Comment:
do we need to revisit naming conventions, because this inherits from "Service" now, not "Provider"; so we should clarify what a "Provider" is and how you tell

Reply:
I think we should drop the "Provider" noun